templates/layouts/bootstrap.html.ep

```diff
@@ -37,29 +37,96 @@
 
   </head>
   <body>
-    <div id="wrapper">
-      %= include_branding 'navbar'
-      <div class='container'>
-        <div class='row' id='subheader'>
-          <div class='col-md-8'>
-            %= breadcrumbs
-          </div>
-          <div class='col-md-4 text-right'>
-            <div id="user-info">
-              %= include 'layouts/subheader_user'
-            </div>
+      <nav class="navbar navbar-static-top">
+          <div class="container">
+              <!-- Brand and toggle get grouped for better mobile display -->
+              <div class="navbar-header">
+                  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-1" aria-expanded="false">
+                      <span class="sr-only">Toggle navigation</span>
+                      <span class="icon-bar"></span>
+                      <span class="icon-bar"></span>
+                      <span class="icon-bar"></span>
+                  </button>
+                  <a class="navbar-brand" href="/"><img src="/images/logo.svg" alt="openQA"></a>
+              </div>
+              <!-- Collect the nav links, forms, and other content for toggling -->
+              <div class="collapse navbar-collapse" id="navbar-collapse-1">
+                  <ul class="nav navbar-nav">
+
+                      <li>
+                          %= link_to 'Jobs' => url_for('tests')
```

Comment:
It used to be 'All Test Results' on / - I only changed it because @sysrich claimed /tests also shows scheduled jobs without results. So 'All Tests' should work.